doc/man3/OSSL_DESERIALIZER_from_bio.pod

```diff
@@ -122,17 +130,28 @@ description above.
 OSSL_DESERIALIZER_CTX_num_deserializers() gets the number of
 deserializers currently added to the context I<ctx>.
 
-OSSL_DESERIALIZER_CTX_set_finalizer() sets the I<finalizer> function
-together with the caller argument for the finalizer, I<finalize_arg>,
-as well as I<cleaner>, the function to clean up I<finalize_arg> when
-the deserialization has concluded.
+OSSL_DESERIALIZER_CTX_set_construct() sets the constructor I<construct>.
 
-OSSL_DESERIALIZER_export() is a fallback function for I<finalizers>
-that can't use the data they get directly for diverse reasons.  It
-takes the same deserialize instance I<deser_inst> that the
-I<finalizer> got and an object I<reference>, unpacks the object that
-refers to, and exports it by creating an L<OSSL_PARAM(3)> array that
-it then passes to I<export_cb>, along with I<export_arg>.
+OSSL_DESERIALIZER_CTX_set_construct_data() sets the constructor data that is
+passed the constructor every time it's called.
+
+OSSL_DESERIALIZER_CTX_set_cleanup() sets the constructor data I<cleanup>
+function.  This is called by L<OSSL_DESERIALIZER_CTX_free(3)>.
+
+OSSL_DESERIALIZER_CTX_get_construct(),
+OSSL_DESERIALIZER_CTX_get_construct_data() and
+OSSL_DESERIALIZER_CTX_get_cleanup()
+return the values that have been set by
+OSSL_DESERIALIZER_CTX_set_construct(),
+OSSL_DESERIALIZER_CTX_set_construct_data() and
+OSSL_DESERIALIZER_CTX_set_cleanup(), respectively.
+
+OSSL_DESERIALIZER_export() is a fallback function for constructors that
+can't use the data they get directly for diverse reasons.  It takes the same
```

Comment:
`s/can't/cannot/`

Reply:
I don't see why...  perhaps my English is thwarted by impressions I've had along the way of learning, I dunno, but "cannot" has been used around me to give off a stronger "not"...  make it more "imperative" if you will, having the use approach "must not", which is absolutely not what I try to express here.

Oy, subtleties of natural languages...

Reply:
To me _can't_ reads informally with the same meaning.  Almost sloppy in documentation.

Another option would be _can not_.

Reply:
I may have a different view of whether documentation needs formal writing. I would see such a requirement as a bit snobbish...

Reply:
The ratio of _cannot_ to _can't_ in the docs is over 8:1.  I think that's a clear preference.

I'll note that documentation is not written informally.  E.g. the insistence on gender neutral pronouns, preferred methods of spelling.

Reply:
Er, we have not actually decided whether our manuals should be written in formal (contractions allowed) or informal style (no contractions). Different people have silently done it differently (I tend to write with contractions, others... I have frankly not really cared to take note). 

That we have chosen to spell certain words correctly, or to use gender neutral language, sure, those are decision points. How that automatically means formal writing all the way, I do not see.